src/locales/ko/mystery-encounters/teleporting-hijinks-dialogue.json

```
@@ -1,27 +1,27 @@
 {
-  "intro": "It's a strange machine, whirring noisily...",
-  "title": "Teleportating Hijinks",
-  "description": "The machine has a sign on it that reads:\n  \"To use, insert money then step into the capsule.\"\n\nPerhaps it can transport you somewhere...",
-  "query": "What will you do?",
+  "intro": "시끄럽게 윙윙 소리가 나는 이상한 기계를 발견했다….",
+  "title": "텔레포트 대소동",
+  "description": "기계에는 다음과 같은 표지판이 있습니다:\n  \"사용하려면, 돈을 넣고 캡슐에 들어가세요.\"\n\n어딘가로 당신을 데려다줄 수 있을겁니다…….",
```

```suggestion
  "description": "기계에는 다음과 같이 설명이 쓰여 있습니다.\n  \"사용하려면 돈을 넣고 캡슐에 들어가세요.\"\n\n기계를 작동시키면 어딘가로 이동할 수 있을 듯하다……",
```
1) 한국어에서는 영어에서처럼 콜론이 사용되지 않다 보니 온점으로 수정했습니다.
2) '표지판'이라는 말이 틀린 건 아니지만, 뭔가 표지판 하면 길이나 상점 같은 데 붙어 있는 큼지막한 판이 먼저 떠오르다 보니 그냥 '설명'이라고 하는 게 자연스럽지 않을까 싶습니다.
3. 'Perhaps it can transport you somewhere' 부분은 기계 설명 부분이 아니라 일반 스크립트라서 하다체로 수정 + 문장을 조금 더 자연스럽게 고쳐 보았습니다.

---

3. event description은 설명이라서 ~합니다 로 통일하기로 했어요!

---

```suggestion
  "description": "기계에는 다음과 같이 설명이 쓰여 있습니다.\n  \"사용하려면 돈을 넣고 캡슐에 들어가세요.\"\n\n어딘가로 당신을 데려다줄 수 있을겁니다…….",
```